app/(app)/settings/index.tsx

```diff
@@ -32,6 +35,16 @@ const SettingsPage = () => {
     [],
   )
 
+  const { mutate, isPending } = useMutation({
+    mutationFn: () => clientApi.usersControllerDeleteUser(),
+    onSuccess: async (response) => {
+      if (response.data) {
+        const responsee = await Auth.deleteUser()
+        if (responsee === 'SUCCESS') await signOut()
```

Comment:
Please use braces

Reply:
As I understand, we still have to implement deleting on Cognito? Please add a TODO comment.

Reply:
Ah, yes, I just didn't see it :D